bindings/python/iota_sdk/types/issuance.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2024 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import List, Optional
+from dataclasses import dataclass, field
+from dataclasses_json import config
+from iota_sdk.types.common import HexStr, json, SlotIndex
+from iota_sdk.types.slot import SlotCommitment
+
+
+@json
+@dataclass
+class IssuanceBlockHeader:
+    """Information that is ideal for attaching a block in the network.
+    Response of GET /api/core/v3/blocks/issuance
+
+    Attributes:
+        strong_parents: Blocks that are strongly directly approved.
+        weak_parents: Blocks that are weakly directly approved.
+        shallow_like_parents: Blocks that are directly referenced to adjust opinion.
+        latest_parent_block_issuing_time: Latest issuing time of the returned parents.
+        latest_finalized_slot: The slot index of the latest finalized slot.
+        latest_commitment: The latest slot commitment.
+    """
+    strong_parents: List[HexStr]
+    weak_parents: Optional[List[HexStr]]
+    shallow_like_parents: Optional[List[HexStr]]
```

Comment:
This is working without `= None`?

Reply:
oh, the linter didn't complain, but that doesn't say much in Python, I guess....so I probably just forgot to add it.

Reply:
Well maybe it's fine if it always gets explicitly set, but I think it could fail if the field is just missing in the JSON and we deserialize then (best to run an example/test which calls this function to see if it works)

Reply:
doesn't work without ... also added a test